### ISSUE TITLE
cimDiablo consistent with cim

### ISSUE THREAD

**Issue by aljabadi** [mixOmicsTeam/mixOmics#94]
cimDiablo to take 'cluster' arg from `cim` and provide similar output.

**Comment by aljabadi** [comment]
Should be implemented in d1659dc5fd0ce22186d233c8be63471d3bd5172d
```r
library(mixOmics)
data(nutrimouse)
Y = nutrimouse$diet
data = list(gene = nutrimouse$gene, lipid = nutrimouse$lipid)
design = matrix(c(0,1,1,1,0,1,1,1,0), ncol = 3, nrow = 3, byrow = TRUE)


nutrimouse.sgccda <- block.splsda(X = data,
                                  Y = Y,
                                  design = design,
                                  keepX = list(gene = c(10,10), lipid = c(15,15)),
                                  ncomp = 2,
                                  scheme = "centroid")

par(mar=c(1,1,1,1)) ## adjust the margins for plot so hopefully no RStudio error
## if you got plot.new() error with the following, manually expand your plot area, remove previous plots and run: graphics.off()
cimDiablo(nutrimouse.sgccda, cluster= 'row') ## cluster rows only
res <- cimDiablo(nutrimouse.sgccda) ## save the output
res$ddr
```

**Comment by aljabadi** [comment]
closed in f9c72b6ba1650674db776a700ee0e551b984349e
